src/client/renderer/r_draw.c

```diff
@@ -301,7 +301,7 @@ static void R_DrawChars(void) {
 		R_BindArray(GL_TEXTURE_COORD_ARRAY, GL_FLOAT, chars->texcoords);
 		R_BindArray(GL_VERTEX_ARRAY, GL_SHORT, chars->verts);
 
-		glDrawArrays(GL_QUADS, 0, chars->vert_index / 2);
+		R_DrawArrays(GL_QUADS, 0, chars->vert_index / 2);
```

Comment:
Also it won't work for these anyways - TRIANGLE_FAN will only work in spots where GL_POLYGON would work. GL_QUADS will eventually have to be changed into GL_TRIANGLES. Sooooup